job.go

```diff
@@ -446,6 +446,16 @@ func (j *Job) ScheduledTime() time.Time {
 	return j.nextRun
 }
 
+// ScheduledUnit returns the scheduled unit of the Job.
+func (j *Job) ScheduledUnit() schedulingUnit {
```

Comment:
there should be a public version of schedulingUnit - or perhaps just make `schedulingUnit` and all the const values public